### ISSUE TITLE
encoding of opendata.md

### ISSUE THREAD

**Issue by ashander** [ropensci/opendata#25]
I'm not sure if this is really an issue, but in pushing #24, my diff ended up with lines like this:

``` bash
--   [neotoma](https://github.com/ropensci/neotoma) (not on CRAN): Programmatic R interface to the Neotoma Paleoecological Database. [Source on GitHub](https://github.com/ropensci/neotoma)
+-   <pkg>neotoma</pkg>: Programmatic R interface to the Neotoma Paleoecological Database. [Source on GitHub](https://github.com/ropensci/neotoma)^M
```

I'm not sure if I induced the DOS encoding somehow, or if my editor was just staying consistent with previous encoding from @leeper

This may be worth mentioning in contrib guidelines

**Comment by leeper** [comment]
Hmm...weird. Might have been me. I was having trouble with Make when I committed, so maybe there was some implicit encoding issue introduced that I didn't notice.

**Comment by ashander** [comment]
Yes, looks like the initial commit of opendata.md did have this encoding:

``` sh
~/code/opendata (master)$ git show 9bb6 -- opendata.md
commit 9bb682e0827b78a48b06a9844d6204b873dcc848
Author: leeper <thosjleeper@gmail.com>
Date:   Sat May 2 17:19:04 2015 +0200

    implement md contribution workflow (closes #17)

diff --git a/opendata.md b/opendata.md
new file mode 100644
index 0000000..fc0451f
--- /dev/null
+++ b/opendata.md
@@ -0,0 +1,356 @@
+This task view contains information about using R to obtain, parse, manipulate, create, and share open data. The focus here is on data discovery, data archiving, open data available in R packages, and packages designed for working with specific types of open data. Much open data is available on the web, and the [WebTechnologies](http://cran.r-project.org/web/views/WebTechnologies.html) TaskView addresses how to obtain and parse web-based data. There is obvious overlap between the two TaskViews, so some packages are described on both.^M
+^M
+Another key issue in a data-focused TaskView is the meaning of "open" data. This TaskView covers many types of data that come with varying degrees of usage restrictions from public domain (or CC-0) data that is useable for any purpose to "freely available" data that is available at no cost but may have licenses that are not strictly speaking "open". Users should investigate the terms of use and licensing of any data referenced here before using it for any particular application.^M
+^M
```

**Comment by ashander** [comment]
.gitattributes specifies a way of dealing with related issue of line endings: 

https://help.github.com/articles/dealing-with-line-endings/#platform-all

I'll close this for now. We can revisit if this crops up in the future (e.g. if someone contributes a patch that doesn't respect the encoding)
